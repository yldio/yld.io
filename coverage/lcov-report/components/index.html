<!doctype html>
<html lang="en">
<head>
    <title>Code coverage report for components</title>
    <meta charset="utf-8" />
    <link rel="stylesheet" href="../prettify.css" />
    <link rel="stylesheet" href="../base.css" />
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <style type='text/css'>
        .coverage-summary .sorter {
            background-image: url(../sort-arrow-sprite.png);
        }
    </style>
</head>
<body>
<div class='wrapper'>
  <div class='pad1'>
    <h1>
      <a href="../index.html">All files</a> components
    </h1>
    <div class='clearfix'>
      <div class='fl pad1y space-right2'>
        <span class="strong">85.06% </span>
        <span class="quiet">Statements</span>
        <span class='fraction'>74/87</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">64.44% </span>
        <span class="quiet">Branches</span>
        <span class='fraction'>29/45</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">73.53% </span>
        <span class="quiet">Functions</span>
        <span class='fraction'>25/34</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">91.18% </span>
        <span class="quiet">Lines</span>
        <span class='fraction'>62/68</span>
      </div>
    </div>
    <p class="quiet">
      Press <em>n</em> or <em>j</em> to go to the next uncovered block, <em>b</em>, <em>p</em> or <em>k</em> for the previous block.
    </p>
  </div>
  <div class='status-line high'></div>
<div class="pad1">
<table class="coverage-summary">
<thead>
<tr>
   <th data-col="file" data-fmt="html" data-html="true" class="file">File</th>
   <th data-col="pic" data-type="number" data-fmt="html" data-html="true" class="pic"></th>
   <th data-col="statements" data-type="number" data-fmt="pct" class="pct">Statements</th>
   <th data-col="statements_raw" data-type="number" data-fmt="html" class="abs"></th>
   <th data-col="branches" data-type="number" data-fmt="pct" class="pct">Branches</th>
   <th data-col="branches_raw" data-type="number" data-fmt="html" class="abs"></th>
   <th data-col="functions" data-type="number" data-fmt="pct" class="pct">Functions</th>
   <th data-col="functions_raw" data-type="number" data-fmt="html" class="abs"></th>
   <th data-col="lines" data-type="number" data-fmt="pct" class="pct">Lines</th>
   <th data-col="lines_raw" data-type="number" data-fmt="html" class="abs"></th>
</tr>
</thead>
<tbody><tr>
	<td class="file high" data-value="grid.js"><a href="grid.js.html">grid.js</a></td>
	<td data-value="85.06" class="pic high"><div class="chart"><div class="cover-fill" style="width: 85%;"></div><div class="cover-empty" style="width:15%;"></div></div></td>
	<td data-value="85.06" class="pct high">85.06%</td>
	<td data-value="87" class="abs high">74/87</td>
	<td data-value="64.44" class="pct medium">64.44%</td>
	<td data-value="45" class="abs medium">29/45</td>
	<td data-value="73.53" class="pct medium">73.53%</td>
	<td data-value="34" class="abs medium">25/34</td>
	<td data-value="91.18" class="pct high">91.18%</td>
	<td data-value="68" class="abs high">62/68</td>
	</tr>

</tbody>
</table>
</div><div class='push'></div><!-- for sticky footer -->
</div><!-- /wrapper -->
<div class='footer quiet pad2 space-top1 center small'>
  Code coverage
  generated by <a href="https://istanbul.js.org/" target="_blank">istanbul</a> at Tue Jul 30 2019 14:23:45 GMT+0100 (WEST)
</div>
</div>
<script src="../prettify.js"></script>
<script>
window.onload = function () {
        if (typeof prettyPrint === 'function') {
            prettyPrint();
        }
};
</script>
<script src="../sorter.js"></script>
<script src="../block-navigation.js"></script>
</body>
</html>
