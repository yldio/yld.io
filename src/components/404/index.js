export * from './Ui';
export { default } from './Page404';
