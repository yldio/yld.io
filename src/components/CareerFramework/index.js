export { default as Discipline } from './Discipline'
