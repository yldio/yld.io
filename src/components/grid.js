import React from 'react'
import styled from 'styled-components'
import breakpoint from 'styled-components-breakpoint'
import remcalc from 'remcalc'
import { spacing, breakpoints } from '../utils/theme'

const css = props => {
  const breaks = [null, ...Object.values(breakpoints)]
  const sx = stylers()
  const rules = []

  for (let key in props) {
    const val = props[key]
    const cx = createRule(breaks, sx)(key, val)

    rules.push(cx)
  }

  return rules.map(r => Array.from(new Set(r)))
}

const createRule = (breaks, sx) => (key, val) => {
  const k = key.charAt(0)
  const style = sx[key] || sx[k] || null

  return toArr(val)
    .map((v, i) => {
      if (style) {
        const bp = breaks[i]
        const decs = style(key, v)
        const body = `${decs};`
        const rule = media(bp, body)

        return rule
      }

      return null
    })
    .filter(r => r !== null)
}

const toArr = n => (Array.isArray(n) ? n : [n])
const num = n => typeof n === 'number' && !isNaN(n)

const dec = args => args.join(':')
const rule = args => args.join(';')
const media = (bp, body) =>
  bp ? `@media screen and (min-width:${remcalc(bp)}){${body}}` : body

const width = scale => (key, n) => {
  if (n === 0) {
    return `
      width: 0;
      position: absolute;
      top: -9999px;
      left: -9999px;
    `
  }

  return `
    ${dec(['width', !num(n) || n > 1 ? px(n) : n * 100 + '%'])}
      position: initial;
      top: 0;
      left: 0;
  `
}
const px = n => (num(n) ? n + 'px' : n)

const space = scale => (key, n) => {
  const [a, b] = key.split('')
  const prop = a === 'm' ? 'margin' : 'padding'
  const dirs = directions[b] || ['']
  const neg = n < 0 ? '-' : ''
  const val = !num(n) ? n : neg + (scale[Math.abs(n)] || Math.abs(n))
  const rules = rule(dirs.map(d => dec([prop + d, val])))

  return rules
}

const directions = {
  t: ['-top'],
  r: ['-right'],
  b: ['-bottom'],
  l: ['-left'],
  x: ['-left', '-right'],
  y: ['-top', '-bottom']
}

const block = (key, n) => dec(['display', n ? 'block' : 'flex'])
const wrap = (key, n) => dec(['flex-wrap', n ? 'wrap' : 'nowrap'])
const auto = (key, n) => dec(['flex', '1 1 auto'])
const column = (key, n) => dec(['flex-direction', n ? 'column' : 'row'])
const align = (key, n) => dec(['align-items', n])
const justify = (key, n) => dec(['justify-content', n])
const order = (key, n) => dec(['order', n])

const stylers = () => ({
  m: space(spacing),
  p: space(spacing),
  width: width(spacing),
  block,
  wrap,
  auto,
  column,
  align,
  justify,
  order
})

const GridStyled = styled.div`
  ${breakpoint('phone')`
    max-width: calc(100% - 62px);

    p {
      max-width: 390px;
    }
  `}
    ${breakpoint('largePhone')`
    max-width: 480px;
    p {
      max-width: 390px;
    }
  `}
   ${breakpoint('smallTablet')`
    max-width: calc(100% - 84px);
 `}
   ${breakpoint('tablet')`
    max-width: calc(100% - 92px);
 `}
   ${breakpoint('desktop')`
    max-width: 1100px;
  `}

    max-width: calc(100% - 48px);
  position: relative;

  ${props => css(props)}
`

const Flex = styled.div`
  flex-wrap: wrap;
  position: relative;
  overflow: hidden;
  ${props => css(props)};
  display: flex;
  ${props =>
    props.block &&
    `
    display: block;
  `};
`

export const Grid = props => <GridStyled {...props} mx="auto" />

export const Row = props => (
  <Flex {...props} mx={[-2, -3, -60, -42, -48, -48]} />
)

export const Col = props => (
  <Flex block {...props} px={[2, 3, 60, 42, 48, 48]} />
)
