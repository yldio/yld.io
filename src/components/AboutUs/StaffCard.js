import React from 'react'
import styled from 'styled-components'
import remcalc from 'remcalc'
import { Col } from '../grid'
import { Padding } from 'styled-components-spacing'
import { BodyPrimary, Subtitle } from '../Typography'

import ExternalAnchor from '../Common/ExternalAnchor'
import Image from '../Common/Image'

const IMAGE_SIZE = 24

const TappableAnchor = styled(ExternalAnchor)`
  width: ${props => remcalc(props.theme.elementSizes.tappableArea)};
  height: ${props => remcalc(props.theme.elementSizes.tappableArea)};
  display: flex;
  justify-content: center;
  align-items: center;

  &:first-child {
    margin-left: -${props => remcalc((props.theme.elementSizes.tappableArea - IMAGE_SIZE) / 2)};
  }
`

const LinksContainer = styled.div`
  display: flex;
  margin-top: -${props => remcalc((props.theme.elementSizes.tappableArea - IMAGE_SIZE) / 2)};
  padding-bottom: ${props =>
    remcalc(18 - (props.theme.elementSizes.tappableArea - IMAGE_SIZE) / 2)};
`

const Description = styled(BodyPrimary)`
  padding-bottom: ${props => remcalc(30)};
`

const SocialLink = ({ name, url, image }) => {
  const size = `${IMAGE_SIZE}px`

  return (
    <TappableAnchor href={url}>
      <Image image={image} alt={name} width={size} height={size} />
    </TappableAnchor>
  )
}

const SocialLinks = ({ data }) => {
  return (data || []).length > 0 ? (
    <LinksContainer>
      {data.map(({ name, url, image }, idx) => (
        <SocialLink
          key={`${name}-${idx}`}
          name={name}
          url={url}
          image={image}
        />
      ))}
    </LinksContainer>
  ) : null
}

const StaffCard = ({ name, image, role, description, socialLinks = [] }) => {
  return (
    <Col width={[1, 1, 1, 1, 6 / 12, 4 / 12]}>
      <Padding bottom={{ smallPhone: 3, smallTablet: 3.5, tablet: 5 }}>
        <Padding bottom={2}>
          <Image image={image} width="100%" />
        </Padding>
        <Subtitle noPadding>{name}</Subtitle>
        <Padding bottom={3}>
          <Subtitle noPadding muted>
            {role}
          </Subtitle>
        </Padding>
        <Description noPadding>{description}</Description>
        <SocialLinks data={socialLinks} />
      </Padding>
    </Col>
  )
}

export default StaffCard
