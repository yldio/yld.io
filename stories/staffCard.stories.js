import React from 'react'
import { storiesOf, addDecorator } from '@storybook/react'
import Theme from './theme'
import { Grid, Row } from '../src/components/grid'
import StaffCard from '../src/components/AboutUs/StaffCard'
import logo from '../src/images/favicon.png'
import twitter from '../src/images/twiter-icon.svg'
import linkedin from '../src/images/linkedin-icon.svg'

addDecorator(Theme)

storiesOf('About Us Page Components', module)
  .add('StaffCard', () => {
    return (
      <Grid>
        <Row>
          <StaffCard
            image={{
              file: {
                url: logo
              }
            }}
            name="Nuno Job"
            role="Chief Executive Officer"
            description="Nuno is the founder and CEO of YLD Group. Previously he was the Chief Commercial Officer at Nodejitsu, where he was responsible for the world's largest Node.js cloud, providing extensive contributions to the success of Node.js. Nuno's formative work years were spent in the USA at IBM Research and MarkLogic. He is also a proud Stanford alumni."
            socialLinks={[
              {
                name: 'twitter',
                image: { file: { url: twitter } },
                url: 'twitter.com/etc'
              },
              {
                name: 'linkedin',
                image: { file: { url: linkedin } },
                url: 'linkedin.com/etc'
              }
            ]}
          />
        </Row>
      </Grid>
    )
  })
  .add('Multiple StaffCards', () => {
    const Sc = (
      <StaffCard
        image={{
          file: {
            url: logo
          }
        }}
        name="Nuno Job"
        role="Chief Executive Officer"
        description="Nuno is the founder and CEO of YLD Group. Previously he was the Chief Commercial Officer at Nodejitsu, where he was responsible for the world's largest Node.js cloud, providing extensive contributions to the success of Node.js. Nuno's formative work years were spent in the USA at IBM Research and MarkLogic. He is also a proud Stanford alumni."
        socialLinks={[
          {
            name: 'twitter',
            image: { file: { url: twitter } },
            url: 'twitter.com/etc'
          },
          {
            name: 'linkedin',
            image: { file: { url: linkedin } },
            url: 'linkedin.com/etc'
          }
        ]}
      />
    )

    return (
      <Grid>
        <Row>
          {Sc}
          {Sc}
          {Sc}
          {Sc}
          {Sc}
        </Row>
      </Grid>
    )
  })
