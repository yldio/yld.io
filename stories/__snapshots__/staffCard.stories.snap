// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots About Us Page Components Multiple StaffCards 1`] = `
.c4 {
  padding-bottom: 1.5rem;
}

.c3 {
  padding-bottom: 2.25rem;
}

.c7 {
  padding-bottom: 2.25rem;
}

.c0 {
  max-width: calc(100% - 48px);
  position: relative;
  margin-left: auto;
  margin-right: auto;
  padding-left: 0;
  padding-right: 0;
}

.c2 {
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  position: relative;
  overflow: hidden;
  padding-left: 0;
  padding-right: 0;
  width: 100%;
  position: initial;
  top: 0;
  left: 0;
  display: block;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  display: block;
}

.c1 {
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  position: relative;
  overflow: hidden;
  margin-left: 0;
  margin-right: 0;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

.c6 {
  color: #333333;
  font-weight: 400;
  font-size: 1.0625rem;
  line-height: 1.5rem;
  padding: 0.75rem 0;
  padding: 0;
  font-weight: 700;
}

.c8 {
  color: #333333;
  font-weight: 400;
  font-size: 1.0625rem;
  line-height: 1.5rem;
  padding: 0.75rem 0;
  opacity: .5;
  padding: 0;
  font-weight: 700;
}

.c5 {
  width: 100%;
  height: auto;
}

.c12 {
  width: 24px;
  height: 24px;
}

.c11 {
  width: 3rem;
  height: 3rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c11:first-child {
  margin-left: -0.75rem;
}

.c10 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  margin-top: -0.75rem;
  padding-bottom: 0.375rem;
}

.c9 {
  color: #333333;
  font-weight: 400;
  font-size: 1.0625rem;
  line-height: 1.5rem;
  padding: 0.75rem 0;
  padding: 0;
  padding-bottom: 1.875rem;
}

@media (min-width:43.8125em) {
  .c3 {
    padding-bottom: 3.375rem;
  }
}

@media (min-width:56.3125em) {
  .c3 {
    padding-bottom: 6.75rem;
  }
}

@media (min-width:29.4375em) {
  .c0 {
    max-width: calc(100% - 72px);
  }
}

@media (min-width:34.5625em) {
  .c0 {
    max-width: 480px;
  }
}

@media (min-width:43.8125em) {
  .c0 {
    max-width: none;
  }
}

@media (min-width:74.8125em) {
  .c0 {
    max-width: 1100px;
  }
}

@media screen and (min-width:29.4375rem) {
  .c0 {
    padding-left: 0;
    padding-right: 0;
  }
}

@media screen and (min-width:34.5625rem) {
  .c0 {
    padding-left: 0;
    padding-right: 0;
  }
}

@media screen and (min-width:43.8125rem) {
  .c0 {
    padding-left: 2.625rem;
    padding-right: 2.625rem;
  }
}

@media screen and (min-width:56.3125rem) {
  .c0 {
    padding-left: 3rem;
    padding-right: 3rem;
  }
}

@media screen and (min-width:74.8125rem) {
  .c0 {
    padding-left: 0;
    padding-right: 0;
  }
}

@media screen and (min-width:29.4375rem) {
  .c2 {
    padding-left: 0;
    padding-right: 0;
  }
}

@media screen and (min-width:34.5625rem) {
  .c2 {
    padding-left: 0;
    padding-right: 0;
  }
}

@media screen and (min-width:43.8125rem) {
  .c2 {
    padding-left: 1.3125rem;
    padding-right: 1.3125rem;
  }
}

@media screen and (min-width:56.3125rem) {
  .c2 {
    padding-left: 1.5rem;
    padding-right: 1.5rem;
  }
}

@media screen and (min-width:74.8125rem) {
  .c2 {
    padding-left: 1.5rem;
    padding-right: 1.5rem;
  }
}

@media screen and (min-width:29.4375rem) {
  .c2 {
    width: 100%;
    position: initial;
    top: 0;
    left: 0;
  }
}

@media screen and (min-width:34.5625rem) {
  .c2 {
    width: 100%;
    position: initial;
    top: 0;
    left: 0;
  }
}

@media screen and (min-width:43.8125rem) {
  .c2 {
    width: 50%;
    position: initial;
    top: 0;
    left: 0;
  }
}

@media screen and (min-width:56.3125rem) {
  .c2 {
    width: 33.33333333333333%;
    position: initial;
    top: 0;
    left: 0;
  }
}

@media screen and (min-width:29.4375rem) {
  .c1 {
    margin-left: 0;
    margin-right: 0;
  }
}

@media screen and (min-width:34.5625rem) {
  .c1 {
    margin-left: 0;
    margin-right: 0;
  }
}

@media screen and (min-width:43.8125rem) {
  .c1 {
    margin-left: -1.3125rem;
    margin-right: -1.3125rem;
  }
}

@media screen and (min-width:56.3125rem) {
  .c1 {
    margin-left: -1.5rem;
    margin-right: -1.5rem;
  }
}

@media screen and (min-width:74.8125rem) {
  .c1 {
    margin-left: -1.5rem;
    margin-right: -1.5rem;
  }
}

<div
  className="c0"
>
  <div
    className="c1"
  >
    <div
      className="c2"
      width={
        Array [
          1,
          1,
          1,
          1,
          0.5,
          0.3333333333333333,
        ]
      }
    >
      <div
        className="c3"
      >
        <div
          className="c4"
        >
          <img
            className="c5"
            src="test-file-stub"
            width="100%"
          />
        </div>
        <h3
          className="c6"
        >
          Nuno Job
        </h3>
        <div
          className="c7"
        >
          <h3
            className="c8"
            muted={true}
          >
            Chief Executive Officer
          </h3>
        </div>
        <p
          className="c9"
        >
          Nuno is the founder and CEO of YLD Group. Previously he was the Chief Commercial Officer at Nodejitsu, where he was responsible for the world's largest Node.js cloud, providing extensive contributions to the success of Node.js. Nuno's formative work years were spent in the USA at IBM Research and MarkLogic. He is also a proud Stanford alumni.
        </p>
        <div
          className="c10"
        >
          <a
            className="c11"
            href="twitter.com/etc"
            onClick={[Function]}
            rel="noopener noreferrer"
            target="_blank"
            title="twitter account"
          >
            <img
              alt="twitter"
              className="c12"
              height="24px"
              src="test-file-stub"
              width="24px"
            />
          </a>
          <a
            className="c11"
            href="linkedin.com/etc"
            onClick={[Function]}
            rel="noopener noreferrer"
            target="_blank"
            title="linkedin account"
          >
            <img
              alt="linkedin"
              className="c12"
              height="24px"
              src="test-file-stub"
              width="24px"
            />
          </a>
        </div>
      </div>
    </div>
    <div
      className="c2"
      width={
        Array [
          1,
          1,
          1,
          1,
          0.5,
          0.3333333333333333,
        ]
      }
    >
      <div
        className="c3"
      >
        <div
          className="c4"
        >
          <img
            className="c5"
            src="test-file-stub"
            width="100%"
          />
        </div>
        <h3
          className="c6"
        >
          Nuno Job
        </h3>
        <div
          className="c7"
        >
          <h3
            className="c8"
            muted={true}
          >
            Chief Executive Officer
          </h3>
        </div>
        <p
          className="c9"
        >
          Nuno is the founder and CEO of YLD Group. Previously he was the Chief Commercial Officer at Nodejitsu, where he was responsible for the world's largest Node.js cloud, providing extensive contributions to the success of Node.js. Nuno's formative work years were spent in the USA at IBM Research and MarkLogic. He is also a proud Stanford alumni.
        </p>
        <div
          className="c10"
        >
          <a
            className="c11"
            href="twitter.com/etc"
            onClick={[Function]}
            rel="noopener noreferrer"
            target="_blank"
            title="twitter account"
          >
            <img
              alt="twitter"
              className="c12"
              height="24px"
              src="test-file-stub"
              width="24px"
            />
          </a>
          <a
            className="c11"
            href="linkedin.com/etc"
            onClick={[Function]}
            rel="noopener noreferrer"
            target="_blank"
            title="linkedin account"
          >
            <img
              alt="linkedin"
              className="c12"
              height="24px"
              src="test-file-stub"
              width="24px"
            />
          </a>
        </div>
      </div>
    </div>
    <div
      className="c2"
      width={
        Array [
          1,
          1,
          1,
          1,
          0.5,
          0.3333333333333333,
        ]
      }
    >
      <div
        className="c3"
      >
        <div
          className="c4"
        >
          <img
            className="c5"
            src="test-file-stub"
            width="100%"
          />
        </div>
        <h3
          className="c6"
        >
          Nuno Job
        </h3>
        <div
          className="c7"
        >
          <h3
            className="c8"
            muted={true}
          >
            Chief Executive Officer
          </h3>
        </div>
        <p
          className="c9"
        >
          Nuno is the founder and CEO of YLD Group. Previously he was the Chief Commercial Officer at Nodejitsu, where he was responsible for the world's largest Node.js cloud, providing extensive contributions to the success of Node.js. Nuno's formative work years were spent in the USA at IBM Research and MarkLogic. He is also a proud Stanford alumni.
        </p>
        <div
          className="c10"
        >
          <a
            className="c11"
            href="twitter.com/etc"
            onClick={[Function]}
            rel="noopener noreferrer"
            target="_blank"
            title="twitter account"
          >
            <img
              alt="twitter"
              className="c12"
              height="24px"
              src="test-file-stub"
              width="24px"
            />
          </a>
          <a
            className="c11"
            href="linkedin.com/etc"
            onClick={[Function]}
            rel="noopener noreferrer"
            target="_blank"
            title="linkedin account"
          >
            <img
              alt="linkedin"
              className="c12"
              height="24px"
              src="test-file-stub"
              width="24px"
            />
          </a>
        </div>
      </div>
    </div>
    <div
      className="c2"
      width={
        Array [
          1,
          1,
          1,
          1,
          0.5,
          0.3333333333333333,
        ]
      }
    >
      <div
        className="c3"
      >
        <div
          className="c4"
        >
          <img
            className="c5"
            src="test-file-stub"
            width="100%"
          />
        </div>
        <h3
          className="c6"
        >
          Nuno Job
        </h3>
        <div
          className="c7"
        >
          <h3
            className="c8"
            muted={true}
          >
            Chief Executive Officer
          </h3>
        </div>
        <p
          className="c9"
        >
          Nuno is the founder and CEO of YLD Group. Previously he was the Chief Commercial Officer at Nodejitsu, where he was responsible for the world's largest Node.js cloud, providing extensive contributions to the success of Node.js. Nuno's formative work years were spent in the USA at IBM Research and MarkLogic. He is also a proud Stanford alumni.
        </p>
        <div
          className="c10"
        >
          <a
            className="c11"
            href="twitter.com/etc"
            onClick={[Function]}
            rel="noopener noreferrer"
            target="_blank"
            title="twitter account"
          >
            <img
              alt="twitter"
              className="c12"
              height="24px"
              src="test-file-stub"
              width="24px"
            />
          </a>
          <a
            className="c11"
            href="linkedin.com/etc"
            onClick={[Function]}
            rel="noopener noreferrer"
            target="_blank"
            title="linkedin account"
          >
            <img
              alt="linkedin"
              className="c12"
              height="24px"
              src="test-file-stub"
              width="24px"
            />
          </a>
        </div>
      </div>
    </div>
    <div
      className="c2"
      width={
        Array [
          1,
          1,
          1,
          1,
          0.5,
          0.3333333333333333,
        ]
      }
    >
      <div
        className="c3"
      >
        <div
          className="c4"
        >
          <img
            className="c5"
            src="test-file-stub"
            width="100%"
          />
        </div>
        <h3
          className="c6"
        >
          Nuno Job
        </h3>
        <div
          className="c7"
        >
          <h3
            className="c8"
            muted={true}
          >
            Chief Executive Officer
          </h3>
        </div>
        <p
          className="c9"
        >
          Nuno is the founder and CEO of YLD Group. Previously he was the Chief Commercial Officer at Nodejitsu, where he was responsible for the world's largest Node.js cloud, providing extensive contributions to the success of Node.js. Nuno's formative work years were spent in the USA at IBM Research and MarkLogic. He is also a proud Stanford alumni.
        </p>
        <div
          className="c10"
        >
          <a
            className="c11"
            href="twitter.com/etc"
            onClick={[Function]}
            rel="noopener noreferrer"
            target="_blank"
            title="twitter account"
          >
            <img
              alt="twitter"
              className="c12"
              height="24px"
              src="test-file-stub"
              width="24px"
            />
          </a>
          <a
            className="c11"
            href="linkedin.com/etc"
            onClick={[Function]}
            rel="noopener noreferrer"
            target="_blank"
            title="linkedin account"
          >
            <img
              alt="linkedin"
              className="c12"
              height="24px"
              src="test-file-stub"
              width="24px"
            />
          </a>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots About Us Page Components StaffCard 1`] = `
.c4 {
  padding-bottom: 1.5rem;
}

.c3 {
  padding-bottom: 2.25rem;
}

.c7 {
  padding-bottom: 2.25rem;
}

.c0 {
  max-width: calc(100% - 48px);
  position: relative;
  margin-left: auto;
  margin-right: auto;
  padding-left: 0;
  padding-right: 0;
}

.c2 {
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  position: relative;
  overflow: hidden;
  padding-left: 0;
  padding-right: 0;
  width: 100%;
  position: initial;
  top: 0;
  left: 0;
  display: block;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  display: block;
}

.c1 {
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  position: relative;
  overflow: hidden;
  margin-left: 0;
  margin-right: 0;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

.c6 {
  color: #333333;
  font-weight: 400;
  font-size: 1.0625rem;
  line-height: 1.5rem;
  padding: 0.75rem 0;
  padding: 0;
  font-weight: 700;
}

.c8 {
  color: #333333;
  font-weight: 400;
  font-size: 1.0625rem;
  line-height: 1.5rem;
  padding: 0.75rem 0;
  opacity: .5;
  padding: 0;
  font-weight: 700;
}

.c5 {
  width: 100%;
  height: auto;
}

.c12 {
  width: 24px;
  height: 24px;
}

.c11 {
  width: 3rem;
  height: 3rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c11:first-child {
  margin-left: -0.75rem;
}

.c10 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  margin-top: -0.75rem;
  padding-bottom: 0.375rem;
}

.c9 {
  color: #333333;
  font-weight: 400;
  font-size: 1.0625rem;
  line-height: 1.5rem;
  padding: 0.75rem 0;
  padding: 0;
  padding-bottom: 1.875rem;
}

@media (min-width:43.8125em) {
  .c3 {
    padding-bottom: 3.375rem;
  }
}

@media (min-width:56.3125em) {
  .c3 {
    padding-bottom: 6.75rem;
  }
}

@media (min-width:29.4375em) {
  .c0 {
    max-width: calc(100% - 72px);
  }
}

@media (min-width:34.5625em) {
  .c0 {
    max-width: 480px;
  }
}

@media (min-width:43.8125em) {
  .c0 {
    max-width: none;
  }
}

@media (min-width:74.8125em) {
  .c0 {
    max-width: 1100px;
  }
}

@media screen and (min-width:29.4375rem) {
  .c0 {
    padding-left: 0;
    padding-right: 0;
  }
}

@media screen and (min-width:34.5625rem) {
  .c0 {
    padding-left: 0;
    padding-right: 0;
  }
}

@media screen and (min-width:43.8125rem) {
  .c0 {
    padding-left: 2.625rem;
    padding-right: 2.625rem;
  }
}

@media screen and (min-width:56.3125rem) {
  .c0 {
    padding-left: 3rem;
    padding-right: 3rem;
  }
}

@media screen and (min-width:74.8125rem) {
  .c0 {
    padding-left: 0;
    padding-right: 0;
  }
}

@media screen and (min-width:29.4375rem) {
  .c2 {
    padding-left: 0;
    padding-right: 0;
  }
}

@media screen and (min-width:34.5625rem) {
  .c2 {
    padding-left: 0;
    padding-right: 0;
  }
}

@media screen and (min-width:43.8125rem) {
  .c2 {
    padding-left: 1.3125rem;
    padding-right: 1.3125rem;
  }
}

@media screen and (min-width:56.3125rem) {
  .c2 {
    padding-left: 1.5rem;
    padding-right: 1.5rem;
  }
}

@media screen and (min-width:74.8125rem) {
  .c2 {
    padding-left: 1.5rem;
    padding-right: 1.5rem;
  }
}

@media screen and (min-width:29.4375rem) {
  .c2 {
    width: 100%;
    position: initial;
    top: 0;
    left: 0;
  }
}

@media screen and (min-width:34.5625rem) {
  .c2 {
    width: 100%;
    position: initial;
    top: 0;
    left: 0;
  }
}

@media screen and (min-width:43.8125rem) {
  .c2 {
    width: 50%;
    position: initial;
    top: 0;
    left: 0;
  }
}

@media screen and (min-width:56.3125rem) {
  .c2 {
    width: 33.33333333333333%;
    position: initial;
    top: 0;
    left: 0;
  }
}

@media screen and (min-width:29.4375rem) {
  .c1 {
    margin-left: 0;
    margin-right: 0;
  }
}

@media screen and (min-width:34.5625rem) {
  .c1 {
    margin-left: 0;
    margin-right: 0;
  }
}

@media screen and (min-width:43.8125rem) {
  .c1 {
    margin-left: -1.3125rem;
    margin-right: -1.3125rem;
  }
}

@media screen and (min-width:56.3125rem) {
  .c1 {
    margin-left: -1.5rem;
    margin-right: -1.5rem;
  }
}

@media screen and (min-width:74.8125rem) {
  .c1 {
    margin-left: -1.5rem;
    margin-right: -1.5rem;
  }
}

<div
  className="c0"
>
  <div
    className="c1"
  >
    <div
      className="c2"
      width={
        Array [
          1,
          1,
          1,
          1,
          0.5,
          0.3333333333333333,
        ]
      }
    >
      <div
        className="c3"
      >
        <div
          className="c4"
        >
          <img
            className="c5"
            src="test-file-stub"
            width="100%"
          />
        </div>
        <h3
          className="c6"
        >
          Nuno Job
        </h3>
        <div
          className="c7"
        >
          <h3
            className="c8"
            muted={true}
          >
            Chief Executive Officer
          </h3>
        </div>
        <p
          className="c9"
        >
          Nuno is the founder and CEO of YLD Group. Previously he was the Chief Commercial Officer at Nodejitsu, where he was responsible for the world's largest Node.js cloud, providing extensive contributions to the success of Node.js. Nuno's formative work years were spent in the USA at IBM Research and MarkLogic. He is also a proud Stanford alumni.
        </p>
        <div
          className="c10"
        >
          <a
            className="c11"
            href="twitter.com/etc"
            onClick={[Function]}
            rel="noopener noreferrer"
            target="_blank"
            title="twitter account"
          >
            <img
              alt="twitter"
              className="c12"
              height="24px"
              src="test-file-stub"
              width="24px"
            />
          </a>
          <a
            className="c11"
            href="linkedin.com/etc"
            onClick={[Function]}
            rel="noopener noreferrer"
            target="_blank"
            title="linkedin account"
          >
            <img
              alt="linkedin"
              className="c12"
              height="24px"
              src="test-file-stub"
              width="24px"
            />
          </a>
        </div>
      </div>
    </div>
  </div>
</div>
`;
